src/gui/protocolwidget.cpp

```diff
@@ -48,7 +48,7 @@ ProtocolWidget::ProtocolWidget(QWidget *parent)
 
     connect(_ui->_tableView, &QTreeWidget::customContextMenuRequested, this, &ProtocolWidget::slotItemContextMenu);
 
-    _model = new ProtocolItemModel(this);
+    _model = new ProtocolItemModel(2000, false, this);
```

Comment:
Shouldn't this be 20000 (so 1 extra zero)?

Reply:
In the protocol the last 2k should be enough with the sync issues however...

Reply:
👍